htscodecs/rANS_static32x16pr.c

```diff
@@ -366,21 +360,44 @@ unsigned char *rans_uncompress_O0_32x16(unsigned char *in,
                 out[i+z+2] = S[2];
                 out[i+z+3] = S[3];
 
-                if (cp < cp_end) {
-                    RansDecRenorm(&R[z+0], &cp);
-                    RansDecRenorm(&R[z+1], &cp);
-                    RansDecRenorm(&R[z+2], &cp);
-                    RansDecRenorm(&R[z+3], &cp);
-                } else {
-                    RansDecRenormSafe(&R[z+0], &cp, cp_end+NX*2);
-                    RansDecRenormSafe(&R[z+1], &cp, cp_end+NX*2);
-                    RansDecRenormSafe(&R[z+2], &cp, cp_end+NX*2);
-                    RansDecRenormSafe(&R[z+3], &cp, cp_end+NX*2);
-                }
+                RansDecRenorm(&R[z+0], &cp);
+                RansDecRenorm(&R[z+1], &cp);
+                RansDecRenorm(&R[z+2], &cp);
+                RansDecRenorm(&R[z+3], &cp);
             }
         }
     }
 
+    // Safe loop
+    for (; i < likely(out_end); i+=NX) {
```

Comment:
Shouldn't this be:
```suggestion
    for (; likely(i < out_end); i+=NX) {
```

Reply:
Oops yes!  Although it's the terminating case when close to the end of the buffer, so is largely irrelevant.  I'll just delete the likely bit.